pandas/io/packers.py

```diff
@@ -115,8 +115,12 @@ def writer(fh):
 
     path_or_buf = _stringify_path(path_or_buf)
     if isinstance(path_or_buf, str):
-        with open(path_or_buf, mode) as fh:
-            writer(fh)
+        try:
+            with open(path_or_buf, mode) as fh:
+                writer(fh)
+        except (FileNotFoundError, ValueError):
```

Comment:
why is there a need to catch ValueError here?

Reply:
I was also thinking that when I included it but I defaulted to leaving it in as a “just to be safe”. 

I’m all for removing it though if it’s unnecessary.